taipy/core/scenario/_scenario_manager.py

```diff
@@ -261,6 +261,17 @@ def _get_all_by_cycle(cls, cycle: Cycle) -> List[Scenario]:
             fil.update({"cycle": cycle.id})
         return cls._get_all_by(filters)
 
+    @classmethod
+    def _get_scenarios_by_time(cls, Year: int, Month: int, Day: int, Hour: int, Minute: int) -> List[Scenario]:
+        duration = datetime.datetime(Year, Month, Day, Hour, Minute)
```

Comment:
What does the `duration` name mean?

Reply:
It's the specific date at which we want to get all it's occuring scenarios.